### ISSUE TITLE
did anybody tried to load the pretrained model provided? 

### ISSUE THREAD

**Issue by adisri2694** [EdinburghNLP/Refresh#21]

**Comment by adisri2694** [comment]
i tried this code
with tf.Session() as sess:
    model = MY_Model(sess, len(vocab_embed_object.vocab_dict)-2)
    selected_modelpath="./pretrained/model.ckpt.epoch-11"
    print("Reading model parameters from %s" % selected_modelpath)
    model.saver.restore(sess, selected_modelpath)
    print("Model loaded.")

got this error 

 Unable to open table file ./pretrained/model.ckpt.epoch-11: Data loss: block checksum mismatch: perhaps your file is in a different file format and you need to use a different restore operator?


using python 3 and tensorflow 1.12

**Comment by adisri2694** [comment]
given that the pretrained model was trained in earlier tensorflow version which is not available now can anybody suggest something to make it work in the new version ?

**Comment by ShirleyHan6** [comment]
You could use the author's code for loading the model. It works for me with python2 and tensorflow 0.10.0

**Comment by adisri2694** [comment]
I used the code that author wrote , can you send me the code that you have
written ,it will be of great help, i have been stuck on this for so long
Thanks

**Comment by ShirleyHan6** [comment]
I did used the author's code and only changed the directory name for the pre-trained model folder etc. 

Are you running this on Windows? It seems that weird errors could occur with tensorflow on Windows. 
Can you try to create a virtual environment then install the dependencies.

**Comment by ShirleyHan6** [comment]
Can you try tensorflow 0.12 instead of 0.10.0rc0? 

Check this link also: https://stackoverflow.com/questions/41048819/how-to-restore-a-model-by-filename-in-tensorflow-r12

**Comment by adisri2694** [comment]
turns out that the model file was not uploaded correctly on the instance,worked fine after reuploading .Thanks for the help

**Comment by shashiongithub** [comment]
Thanks for helping one another guys. I have been a bit busy with the move.
